os_net_config/impl_nmstate.py

```diff
@@ -471,7 +476,7 @@ def set_ifaces(self, iface_data):
         :param verify: boolean that determines if config will be verified
         """
         state = {Interface.KEY: iface_data}
-        self.__dump_config(state, msg=f"Overall interface config")
+        self.__dump_config(state, msg=f"Prepared interface config")
```

Comment:
Comments needs to be updated for:
   - rules_state (no input "name")
   - set_ifaces (no input "verify", return "state"
   - set_dns

Reply:
Done